TACC/protx-dashboard
# Task/add captions to plots

## Overview: ##
Add annotation text for analytics charts.

## Related Jira tickets: ##

* [COOKS-123](https://jira.tacc.utexas.edu/browse/COOKS-123)

## Summary of Changes: ##
New FigureCaption component and corresponding css file were added.

## Testing Steps: ##
1. Go to https://cep.test/protx/dash/analytics
2. Check the annotation text under the chart.
3. Click on a county and check the annotation text under the chart.

## UI Photos:
![Screen Shot 2022-10-21 at 2 25 39 PM](https://user-images.githubusercontent.com/79928051/197275107-dbe65fe3-e3ef-45d9-ab15-8a0d065a5d0c.png)
![Screen Shot 2022-10-21 at 2 26 37 PM](https://user-images.githubusercontent.com/79928051/197275120-90ce1e74-938f-4b7c-96bf-dd613f03815b.png)

## Notes: ##
The new component, FigureCaption, will need to be refactored at some point. I encountered some issues when passing html markup with the text to the component for the county chart annotation.
